astroquery/mast/tests/test_mast_remote.py

```diff
@@ -1088,3 +1088,71 @@ def test_zcut_get_cutouts(self):
         assert isinstance(cutout_list, list)
         assert len(cutout_list) == 1
         assert isinstance(cutout_list[0], fits.HDUList)
+
+    ###################
+    # HapcutClass tests #
+    ###################
+
+    def test_hapcut_download_cutouts(self, tmpdir):
+        
+        # Test 1: Simple API call with expected results
+        coord = SkyCoord(351.347812, 28.497808, unit="deg")
+
+        cutout_table = mast.Hapcut.download_cutouts(coordinates=coord, size=5, path=str(tmpdir))
+        assert isinstance(cutout_table, Table)
+        assert len(cutout_table) >= 1
+        for row in cutout_table:
+            assert os.path.isfile(row['Local Path'])
+            if 'fits' in os.path.basename(row['Local Path']):
+                assert fits.getdata(row['Local Path']).shape == (5, 5)
```

Comment:
could we check the shape for the other size inputs, too?

Reply:
Thanks you just helped me catch a bug. Updated.